tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -22,8 +22,10 @@ namespace tensorflow {
 
 namespace {
 
-// In case of failure, every call will be retried kMaxAttempts-1 times.
-constexpr int kMaxAttempts = 4;
+// In case of failure, every call will be retried kMaxAttempts times.
+constexpr int kMaxAttempts = 3;
```

Comment:
I see how this can be an issue due to the how the variable is named. 

Personally I prefer the first approach. Although it involves more code change, it is easier to read as retry count makes everything slightly more readable.